Source/ui_win32/Res.rc

```diff
@@ -157,6 +169,8 @@ BEGIN
         MENUITEM "State Slot",                  ID_MAIN_VM_STATESLOT
         MENUITEM "Save State\tF7",              ID_MAIN_VM_SAVESTATE
         MENUITEM "Load State\tF8",              ID_MAIN_VM_LOADSTATE
+        MENUITEM SEPARATOR
```

Comment:
These two lines should be the only changes in this file. All other changes to `Res.rc` are unnecessary and should be removed.